src/main/java/lotto/utils/StringAddCalculator.java

```diff
@@ -0,0 +1,76 @@
+package lotto.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static lotto.constants.CalculatorConstants.*;
+
+public class StringAddCalculator {
+
+    private static Pattern pattern = Pattern.compile("//(.)\n(.*)");
+
+    public void stringCalculator(String inputString) {
+        int sum = getNumberList(inputString);
+        System.out.println("합산 : " + sum);
+    }
+
+    public int getNumberList(String inputString) {
+        String args = getCheckInputString(inputString);
+        List<Integer> list = splitInputString(args);
+        return list.stream().mapToInt(Integer::intValue).sum();
```

Comment:
```suggestion
        return list.stream()
                .mapToInt(Integer::intValue)
                .sum();
```
하나의 라인에 하나의 .을 사용하면 가독성이 좋아집니다 :)
반대로 하나의 라인에 .을 사용하지 않은 경우는 사용하도록 변경해주시면 좋을 것 같아요 👍